src/carousel.css.ts

```diff
@@ -112,7 +112,7 @@ export const vars = {
 
 export const DEFAULT_DESKTOP_GAP = 16;
 const DEFAULT_MOBILE_GAP = 8;
-const DEFAULT_MOBILE_PAGE_OFFSET = '16px';
+const DEFAULT_MOBILE_PAGE_OFFSET = '24px';
```

Comment:
as I understood, this was going to change to 40px. is this right?

Reply:
this 24 is for intermediate pages. First and last page has a 40px offset